docs/csharp/whats-new/csharp-7-1.md

```diff
@@ -0,0 +1,176 @@
+---
+title: What's new in C# 7.1
+description: An overview of new features in C# 7.1.
+keywords: C# language design, 7.1, Visual Studio 2017, 
+author: billwagner
+ms.author: wiwagn
+ms.date: 08/16/2017
+ms.topic: article
+ms.prod: .net
+ms.devlang: devlang-csharp
+---
+
+# What's new in C# 7.1
+
+C# 7.1 is the first point release to the C# Language. This release is the
+first step toward the next major release of the langauge. This increased
+cadence enables you to use the new features sooner, ideally when
+each features is ready. This release also adds the ability to configure
+the compiler to match a specified version of the language. This feature
+separates the decision to upgrade tools from the decision to upgrade language
+versions.
+
+This release adds the [language version selectionn](#language-version-selection)
+configuration element, and four new language features.
+
+The new language features in this release are:
+
+* [`async` `Main` method](#async-main)
+  - The entry point for a console application can have the `async` modifier.
+* [`default` value expressions](#default-value-expressions)
+  - The `default(T)` expression may omit the target type when that type can be inferred.
```

Comment:
Right. I re-worked the sentence a bit. Yes, this new feature is "default literal expressions", which apply in the context of "default value expressions". I'm linking to updates to the topic on "default value expressions". Let me know what you think of the revision